pkg/trace/api/otlp_test.go

```diff
@@ -492,6 +517,58 @@ func TestOTLPSetAttributes(t *testing.T) {
 	})
 }
 
+func unflatten(str string) map[string]string {
+	parts := strings.Split(str, ",")
```

Comment:
This seems to suggest container tag values cannot have commas, may be worth documenting

Reply:
This is the standard notation for tag compression. The same type of encoding is used in metrics. I'm sure it _is_ documented somewhere in the metrics pipeline. But good observation. I guess as far as the tests go, this is good enough I'd say.